net/core/dev.c

```diff
@@ -9781,6 +9790,7 @@ static void __net_exit default_device_exit_batch(struct list_head *net_list)
 	 * will run in the rtnl_unlock() at the end of
 	 * default_device_exit_batch.
 	 */
+
```

Comment:
this is spurious right?

Reply:
Yes, sorry about that. Will correct it in the next fix up